### DISCUSSION TITLE
Guidance Needed for Octernships Project Setup

### DISCUSSION THREAD

**Discussion by sidhyaashu** [community/community#146716] [GitHub Education]
### Hi everyone,

Hi everyone,

I hope you're all doing great! I'm currently participating in an Octernship project and need some guidance to get started effectively.

Here’s what I’m facing:
1. I’m unsure how to set up the project environment as per the requirements.
2. The project documentation mentions certain tools and workflows that I’m not familiar with.
3. I’m looking for best practices or tips from others who have successfully completed their Octernship projects.

If anyone has insights or resources they could share, I’d greatly appreciate it! Additionally, are there any common challenges to watch out for during the process?

Thanks in advance for your help! Looking forward to learning from your experiences. 😊

**Comment by upsaurav12** [comment]
i don't think i open currently.
